### PR TITLE
Display app_links always

### PR DESCRIPTION
**Pull request by TiiFuchs**
I figured that app links configured in the backend will not get displayed if the array $MAILCOW_APPS in inc/vars.inc.php or vars.local.inc.php is empty...

I changed it so that $app_links does get displayed even if $MAILCOW_APPS is empty.

### CODE REVIEW

**Comment by andryyy** [comment]
Hi, 

Thanks. 

This adds some unnecessary white spaces. 

There is also the navbars dropdown left.

**Comment by TiiFuchs** [comment]
I just checked: In the navbar it already shows the app_links, if MAILCOW_APPS is empty, doesn't it?

I also can't see any unwanted whitespace. Can you clarify what you mean exactly?

File: data/web/index.php [hunk 1]
```diff
@@ -87,23 +87,23 @@
...
-                ?>
-                  <a href="<?= htmlspecialchars($val); ?>" role="button" style="margin-bottom:3pt" class="btn btn-primary"><?= htmlspecialchars($key); ?></a>&nbsp;
-                <?php 
-                  }
-                }
-              }
-            }
-          }
-          ?>
+              }
```
Comment (mkapra) [inline data/web/index.php]: I think he means whitespaces like behind this bracket.

**Comment by TiiFuchs** [comment]
I finally found the time to remove those trailing spaces from githubs online ide...

**Comment by andryyy** [comment]
Just curious and tired, so I cannot check it myself right now: What happens when both are empty? Links created in the UI and in vars?

**Comment by TiiFuchs** [comment]
It just has the label/legend "mailcow Apps" (or what is configured in the UI modifications in the backend) and an empty space below... (Like before I think...?)

grafik

**Comment by andryyy** [comment]
Okay, thanks. :)

**Pull request merged**